src/test/java/com/hocs/test/pages/complaints/BFProgressCase.java

```diff
@@ -60,6 +65,37 @@ public void createCaseOfTypeAndMoveItToTargetStageWithSpecifiedComplaintType(Str
         moveCaseOfTypeFromCurrentStageToTargetStage(caseType, "N/A", targetStage);
     }
 
+    public void attemptEscalateBFCaseToStage2() throws Exception {
+        dashboard.selectSearchLinkFromMenuBar();
+        search.enterComplaintsSearchCriteria("Complainant Home Office Reference", getCurrentMonth() + "/" + getCurrentYear());
+        search.clickTheButton("Search");
+        search.waitForResultsPage();
+        if (search.checkVisibilityOfEscalationHypertext()) {
+            WebElementFacade bfCaseRefField = findBy("//a[contains(text(), 'Escalate case')]/parent::td/preceding-sibling::td/a");
+            String bfCaseRef = bfCaseRefField.getText();
+            setSessionVariable("complaintsCaseReference").to(bfCaseRef);
```

Comment:
could we change this method variable and session variable to indicate which of the 2 case references this is, the stage1 or stage 2 reference.